.github/workflows/trigger_ado_int_tests.yml

```diff
@@ -161,12 +161,12 @@ jobs:
           # if status is not completed, or the result is one of failed, canceled; then fail the job
           if [[ "$status" != "completed" || "$result" =~ (failed|canceled) ]]; then
             echo "::error::Pipeline did not complete successfully"
-            echo "# ❌ ${PIPELINE:-$DEFAULT_TITLE} Failed ❌" >> $GITHUB_STEP_SUMMARY
+            echo "# [FAILED] ${PIPELINE:-$DEFAULT_TITLE} Failed [FAILED]" >> $GITHUB_STEP_SUMMARY
```

Comment:
Emoji's causing issue ?

Reply:
[Seems like it?](https://github.com/Azure/azure-iot-cli-extension/actions/runs/18706894428/job/53346549375#step:5:226)

```
Error:      2025-10-22 06:04:44,457 [ThreadPoolEx] [WARNI]  Fail to read file .\.github\workflows\trigger_ado_int_tests.yml. error: 'charmap' codec can't decode byte 0x9d in position 5523: character maps to <undefined>```

Reply:
It's probably something we could fix with an SDL config later, but since the emojis are of little value (personal opinion 😄 ) I think this fix works fine for now